src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticationProperties.cs

```diff
@@ -20,27 +20,33 @@ public class AuthenticationProperties
         internal const string UtcDateTimeFormat = "r";
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="AuthenticationProperties"/> class
+        /// Initializes a new instance of the <see cref="AuthenticationProperties"/> class.
         /// </summary>
         public AuthenticationProperties()
             : this(items: null)
         {
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="AuthenticationProperties"/> class
+        /// Initializes a new instance of the <see cref="AuthenticationProperties"/> class.
         /// </summary>
         /// <param name="items"></param>
         public AuthenticationProperties(IDictionary<string, string> items)
```

Comment:
Do we want a copy constructor/overload that lets you easily copy both Items and Parameters into a new AuthenticationProperties?

Reply:
Hmm, I didn’t want to add one for the `Properties` dictionary as that would likely be confusing but a copy constructor could be useful.

Reply:
Yeah either a copy constructor that takes AuthenticationProperties or one that takes items + properties, should have at least one of those options.

Reply:
Make it one that takes both collections, not a copy constructor. We may need this if we have to wrap Identitie's result.

Reply:
The current constructor will not copy the passed `Items` though but will use the passed collection as is. Would you want the same behavior for the `Parameters` dictionary then? This would be kind of the opposite of a copy constructor then.

Reply:
Yes, take the existing collection. It's a very shallow copy 😁